docker/build_docker.sh

```diff
@@ -2,7 +2,8 @@
 set -euo pipefail
 
 image="composable/emulated-light-client"
+tag="$(git describe --tags --exact-match HEAD 2>/dev/null || git rev-parse --short HEAD)"
```

Comment:
I think it should refer to HEAD in master branch tbh.

Reply:
@dhruvja I don't 100% agree with that. Ideally, you want to build the image in the same branch which is currently active. At least, that's how we ship & build images in our CI/CDs and local envs.

Reply:
but there can be some changes in the PRs which can break the validator. So i dont recommend to build from the latest branch.

Reply:
It should be validator's responsibility to check out a proper revision

Reply:
we can merge if you point to the HEAD in master branch.

Reply:
because for example, when u point to head -> it would install from this https://github.com/ComposableFi/emulated-light-client/pull/334 which would break since this is not deployed yet.

Reply:
No, `HEAD` always points to what user has currently checked out from the git repository. Simply put, if user locally checks out `master`, then `HEAD` will point to that; similarly if user checks out #334 branch, `HEAD` will point to that locally.

See [how HEAD works](https://git-scm.com/book/en/v2/Git-Internals-Git-References#ref_the_ref)

Reply:
I see. I was mistaken then. Looks good to me then.